pkg/k8shandler/services.go

```diff
@@ -52,8 +71,12 @@ func createOrUpdateService(serviceName, namespace, clusterName, targetPortName s
 	return nil
 }
 
-func createService(serviceName, namespace, clusterName, targetPortName string, port int32, selector, labels map[string]string, publishNotReady bool) *v1.Service {
+func createService(serviceName, namespace, clusterName,
```

Comment:
was this formatted automatically by `go fmt`? it might make it a little easier to read if we have like-typed args on the same line... 
```
func createService(serviceName, namespace, clusterName targetPortName string,
                               port int32, selector, annotations, labels map[string]string, publishNotReady bool) *v1.Service {
```